graphql_api/views.py

```diff
@@ -320,7 +320,7 @@ def _check_ratelimit(self, request):
             user_ip = self.get_client_ip(request)
             key = f"rl-ip:{user_ip}"
 
-        limit = 300
+        limit = settings.GRAPHQL_RATE_LIMIT_VALUE
         window = 60  # in seconds
```

Comment:
May be nice to have an `GRAPHQL_RATE_LIMIT_ENABLED` knob as well if not too annoying to add in. May be easier for DEC/self-hosted to use than trying to guess a large number for their use case :shrug:

Reply:
Fair point! I like it